ehr_sqlite3.py

```diff
@@ -0,0 +1,207 @@
+        unit: tuple = self.cur.execute(
+            """select LabUnits from Labs where ObsID= ?""",
+            (self.ObsID,),
+        ).fetchone()
+        return unit[0]
+
+    @property
+    def value(self) -> float:
+        """Return lab units."""
+        value: tuple[str] = self.cur.execute(
+            """select LabValue from Labs where ObsID = ?""", (self.ObsID,)
+        ).fetchone()
+        return float(value[0])
+
+    @property
+    def date(self) -> datetime:
+        """Return the date of test."""
+        DOB: str = self.cur.execute(
+            """select LabDateTime from Labs where ObsID = ?""",
+            (self.ObsID,),
+        ).fetchone()[0]
+        DOB_dt: datetime = datetime.strptime(DOB, "%Y-%m-%d %H:%M:%S.%f")
+        return DOB_dt
+
+
+# below we create a patient class that extracts
+# relevant information from the database
+# It also includes information about the Labs
+# attended by the patient. These labs are storing
+# information in form of lab class.
+class Patient:
+    """Store a patient data."""
+
+    def __init__(self, id: str, dbpath: str) -> None:
+        """Instantiate patient class."""
+        self.id: str = id
+        self.dbpath: str = dbpath
+        con = sqlite3.connect(dbpath)
+        self.cur = con.cursor()
+
+    def __str__(self) -> str:
+        """Patient Class Representation."""
+        return self.id + " Class"  # O(1)
+
+    @property
+    def dateofbirth(self) -> datetime:
+        """Extract DOB from database."""
+        DOB: str = self.cur.execute(
+            """select PatientDateofBirth from Patients where PatientID = ?""",
+            (self.id,),
+        ).fetchone()[0]
+        DOB_dt: datetime = datetime.strptime(DOB, "%Y-%m-%d %H:%M:%S.%f")
+        return DOB_dt
+
+    @property
+    def gender(self) -> str:
+        """Extract gender from database."""
+        gen: tuple[str] = self.cur.execute(
+            """select PatientGender from Patients where PatientID = ?""", (self.id,)
+        ).fetchone()
+        return gen[0]
+
+    @property
+    def race(self) -> str:
+        """Extract race from database."""
+        race: tuple[str] = self.cur.execute(
+            """select PatientRace from Patients where PatientID = ?""", (self.id,)
+        ).fetchone()
+        return race[0]
+
+    @property
+    def labs(self) -> list[Lab]:
+        """Exctract all labs attended by patient."""
+        labs: list[tuple[str]] = self.cur.execute(
+            """select ObsID from Labs where PatientID = ?""", (self.id,)
+        ).fetchall()
+
+        labs_cl: list[Lab] = [Lab(i[0], self.dbpath) for i in labs]
+        return labs_cl
+
+
+# This function takes the data and creates a
+# patient class to store them in dictionary.
+# We could have done this in the create
+# data base function above, but we assume for that the
+# user might want to create database only once and hence,
+# provide a seperate function to store information for classes
```

Comment:
Thanks, this is helpful. I'm not sold on the design, though. This depends on the same `patient_path` and `dbpath` being passed to both functions (`create_database` and `store_patient_class`). It would be nice to have just one authoritative data source at any given time. If, when calling `store_patient_class`, you assume that the database is already populated, couldn't you extract the patient ids from the database? Then `store_patient_class` would not depend on `patient_path`.

Also, please add tests for `store_patient_class`.